app/src/components.html

```diff
@@ -1,65 +1,109 @@
 <template>
   <require from="./theming.css"></require>
+  <require from="./components.css"></require>
+
   <section>
     <h1>Components</h1>
     <p>Demo of each component and diverse styles and types</p>
   </section>
 
   <section>
     <h2>Buttons</h2>
-  
+
     <ux-grid class="ux-grid--remove-padding">
       <ux-grid-cell sm="4">
         <h4>Primary (default)</h4>
-        <p><ux-button type="raised">Raised</ux-button></p>
-        <p><ux-button type="flat">Flat</ux-button></p>
-        <p><ux-button type="text">Text</ux-button></p>
-        <p><ux-button type="outline">Outline</ux-button></p>
-        <p><ux-button type="fab">Fab</ux-button></p>
+        <p>
+          <ux-button type="raised">Raised</ux-button>
+        </p>
+        <p>
+          <ux-button type="flat">Flat</ux-button>
+        </p>
+        <p>
+          <ux-button type="text">Text</ux-button>
+        </p>
+        <p>
+          <ux-button type="outline">Outline</ux-button>
+        </p>
+        <p>
+          <ux-button type="fab">Fab</ux-button>
+        </p>
       </ux-grid-cell>
       <ux-grid-cell sm="4">
         <h4>Accent (.ux-button--accent)</h4>
-        <p><ux-button class="ux-button--accent" type="raised">Raised</ux-button></p>
-        <p><ux-button class="ux-button--accent" type="flat">Flat</ux-button></p>
-        <p><ux-button class="ux-button--accent" type="text">Text</ux-button></p>
-        <p><ux-button class="ux-button--accent" type="outline">Outline</ux-button></p>
-        <p><ux-button class="ux-button--accent" type="fab">Fab</ux-button></p>
+        <p>
+          <ux-button class="ux-button--accent" type="raised">Raised</ux-button>
+        </p>
+        <p>
+          <ux-button class="ux-button--accent" type="flat">Flat</ux-button>
+        </p>
+        <p>
+          <ux-button class="ux-button--accent" type="text">Text</ux-button>
+        </p>
+        <p>
+          <ux-button class="ux-button--accent" type="outline">Outline</ux-button>
+        </p>
+        <p>
+          <ux-button class="ux-button--accent" type="fab">Fab</ux-button>
+        </p>
       </ux-grid-cell>
       <ux-grid-cell sm="4">
         <h4>Disabled (disabled)</h4>
-        <p><ux-button disabled type="raised">Raised</ux-button></p>
```

Comment:
Well, there's merit with both type of formatting, so I'd suggest we get @ben-girardet input on this. Once this view gets big, it's not too bad for merging one child block content into a single line.

Reply:
It's Beautify at work. The project probably needs a config checked in if formatting is important.

Reply:
It's not that important, at least for now. But since @ben-girardet creates this, I just didn't want to rock it 😓

Reply:
I don’t think it’s important either. 

However as a best practice we can try to avoid formatting changes in PR as it doesn’t refer to the subject of the PR and make it a little more complicated to review. 

No need to change anything now from my point of view. Dropping this comment as a best practice idea for future commits.